### PR TITLE
Remove the run interface in operations.py

### PR DESCRIPTION
**Pull request by adgnabro**
Deprecated operations.py method ``run`` (#463)

## Description
- Add deprecation decorator into flow/operations.py
- Update contributors.yaml
- Update changelog.txt

## Motivation and Context
Unused interface

## Types of Changes
- [x] Documentation update
- [ ] Bug fix
- [ ] New feature
- [x] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:
- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

File: flow/operations.py [hunk 1]
```diff
@@ -172,7 +173,7 @@ def _get_operations(include_private=False):
             if len(signature.args) == 1:
                 yield name
 
-
+@deprecated(deprecated_in="0.12", removed_in="0.14", current_version=__version__)
```
Comment (b-butler) [inline flow/operations.py:176]: To use `__version__` you need to import it from `.version`.

**Review by b-butler — DISMISSED** [review]
I left one comment about importing `__version__` otherwise looks good to me. Thanks!

**Comment by b-butler** [comment]
@adgnabro the pre-commit checks are failing. See the instructions towards the bottom of our community webpage about installing pre-commit https://docs.signac.io/en/latest/community.html.

File: changelog.txt [hunk 2]
```diff
@@ -61,6 +61,7 @@ Changed
 - Python 3.6 is only tested with oldest dependencies (#436).
 - Drop support for `tqdm <https://github.com/tqdm/tqdm>`__ versions older than `4.48.1` (#436, #440).
 - Drop support for `Jinja2 <https://palletsprojects.com/p/jinja/>`__ versions older than `2.10.0` (#436).
+- Deprecated operations.py method ``run`` (#463).
```
Comment (bdice) [inline changelog.txt:64]: Added a bit more info.
```suggestion
- Deprecated ``operations.py`` method ``flow.run`` (#221, #463, #467).
```

**Review by bdice — APPROVED** [review]
Nice work @adgnabro! Thank you very much for your pull request and welcome to the signac team. 😄 I have one small improvement to the changelog. I will apply that change directly to your PR. Once this PR receives a second approval, it will be ready to merge. Thanks @b-butler for taking a first pass of review.

**Review by kidrahahjo — APPROVED** [review]
Looks good, thank you @adgnabro for this PR.

**Comment by adgnabro** [comment]
You're welcome !

**Pull request merged**